prow/plugins/updateconfig/updateconfig.go

```diff
@@ -123,8 +123,16 @@ func handle(gc githubClient, kc kubeClient, log *logrus.Entry, pre github.PullRe
 		return err
 	}
 
+	message := func(cm plugins.ConfigMapSpec, fileName string) string {
+		msg := fmt.Sprintf("%s configmap in %s", fileName, cm.Name)
```

Comment:
The args are backwards here and the message could use some markdown formatting.
Try ```fmt.Sprintf("`%s` configmap from file `%s`", cm.Name, fileName)```